### ISSUE TITLE
Questions about model size

### ISSUE THREAD

**Issue by appleleaves** [larq/larq#297]
Hi.
I compare the model size output by lq.model.summery() and get the birealnet model size as 4.03 MB which is different from that reported in paper 33.6Mbit=4.3MB.
I wonder why.

**Comment by leonoverweel** [comment]
> Hi @appleleaves, thanks for your question. We've looked into this and our implementation differs slightly from the paper's implementation, which is probably the reason our model is 4.03 MB instead of 4.2 MB (= 33.6 Mbit). 

> You can see this by comparing the Netron diagrams of [our implementation](https://lutzroeder.github.io/netron/?url=https://cors-anywhere.herokuapp.com/https://github.com/larq/zoo/releases/download/birealnet-v0.3.0/birealnet.json) with [the original implementation](https://lutzroeder.github.io/netron/?url=https://cors-anywhere.herokuapp.com/https://raw.githubusercontent.com/liuzechun/Bi-Real-net/398f1406517b2ac109b0455ba2a1e3c37aeb780c/18-layer/Bi-Real-net-18-train.prototxt). Some of the differences you see there are due to notation, such as the original paper's `Threshold` + `Binary Convolution` layers being encapsulated in our single `QuantConv2D` layer, and their `BatchNorm` + `Scaling` layers being encapsulated in our single `BatchNormalization` layer. Other differences, like our extra BatchNormalization every 4th block and the different final layers, could explain the difference in model size.

**Comment by leonoverweel** [comment]
Hi @appleleaves, small update: our model size calculations use 1 MB = 1,024 KB ([JEDEC standard](https://en.wikipedia.org/wiki/JEDEC_memory_standards#Unit_prefixes_for_semiconductor_storage_capacity)), and perhaps the paper used 1 Mbit = 1,000 Kbit. If you use 1 MB = 1,000 KB for our model size, you'd get a model size of 4.22 MB for our implementation (slightly bigger because of our additional batchnorms). Does this answer your question?

**Comment by appleleaves** [comment]
Yes, thanks a lot!
